modules/node/src/appRegistry/appActions.service.ts

```diff
@@ -226,6 +225,7 @@ export class AppActionsService {
     } as HashLockTransferAppAction);
 
     await this.cfCoreService.uninstallApp(senderApp.identityHash);
-    this.log.info(`Reclaimed collateral from ${senderApp.identityHash}`);
+    console.log(`UNINSTALLED SENDER APP AT ${Date.now()}`);
```

Comment:
typo?

Reply:
Ah this is for the test, should be deleted.